kura/test/org.eclipse.kura.cloudconnection.sparkplug.mqtt.provider.test/src/test/java/org/eclipse/kura/cloudconnection/sparkplug/mqtt/transport/test/ConfigurationUpdateTest.java

```diff
@@ -0,0 +1,154 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Eurotech and/or its affiliates and others
+ * 
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ * 
+ * Contributors:
+ *  Eurotech
+ *******************************************************************************/
+package org.eclipse.kura.cloudconnection.sparkplug.mqtt.transport.test;
+
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.anyBoolean;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.net.ssl.SSLContext;
+
+import org.eclipse.kura.cloudconnection.sparkplug.mqtt.transport.SparkplugDataTransport;
+import org.eclipse.kura.cloudconnection.sparkplug.mqtt.transport.SparkplugDataTransportOptions;
+import org.eclipse.kura.data.transport.listener.DataTransportListener;
+import org.eclipse.kura.ssl.SslManagerService;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ConfigurationUpdateTest {
+
+    private SparkplugDataTransport transport = new SparkplugDataTransport();;
+    private DataTransportListener listener = mock(DataTransportListener.class);
+    private SslManagerService sslManagerService = mock(SslManagerService.class);
+    private Exception occurredException;
+
+    /*
+     * Scenarios
+     */
+
+    @Test
+    public void test() {
+        assertTrue(true);
+    }
+
+    @Test
+    public void shouldUpdateConfigurationOnSetSslManager() {
+        givenUpdated();
```

Comment:
Could we extract the config properties from method and pass them as parameter ? In that way the test is more readable and the used configuration is well exposed